examples/podman_local/my_project/README.md

```diff
@@ -0,0 +1,46 @@
+
+# Feast Installation with Podman
```

Comment:
may be feast demo with podman or feast example with podman title makes more sense IMO.

Reply:
done